client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -28,19 +28,52 @@ import (
 	log "github.com/Sirupsen/logrus"
 	dockerClient "github.com/docker/engine-api/client"
 	dockerTypes "github.com/docker/engine-api/types"
+	"github.com/docker/engine-api/types/filters"
 	"github.com/docker/engine-api/types/swarm"
+	"io"
+	"os"
+	"time"
 )
 
 const (
 	// dockerAPIVersion: docker engine 1.24 and above support this api version
 	dockerAPIVersion = "v1.24"
 	// dockerUSocket: Unix socket on which Docker engine is listening
 	dockerUSocket = "unix:///var/run/docker.sock"
+	// Postfix added to names of Samba services for volumes
+	serviceNamePostfix = "fileServer"
+	// Path where the file server image resides in plugin
+	fileServerPath = "/usr/lib/samba.tar"
```

Comment:
OTOH if this is a path inside the container does this path get propagated to Docker for Docker to be able to access the file? Ideally better to keep plugin artifacts outside of access to any user outside of the plugin.